### DISCUSSION TITLE
Cannot upload a file

### DISCUSSION THREAD

**Discussion by Nanouz** [Dotnet-Boxed/Templates#178] [Q&A]
Hi

I am trying to upload a single file in a form like I use to do but I get this error message in my console : 
XML Parsing Error: no root element found Location

If I comment the input file field everything posts perfectly. Is this something related with the template? I never seen this before.

Thx!

**Which project template is the issue about?**
ASP MVC Boilerplate
...

**Which version of the project template is the issue about?**
?
...

**Which version of Visual Studio or the dotnet SDK were you using?**
VS 2015 Update 3
...

**Comment by RehanSaeed** [comment]
I don't really have enough information to give you any things you might be able to try. Can you tell me more about the error? Are you using the API or Website template? Is it MVC 5 or 6?

**Comment by Nanouz** [comment]
I am using the website template and MVC 5.

**Comment by RehanSaeed** [comment]
>  XML Parsing Error: no root element found Location

Suggests that the XML document was empty or invalid.

**Comment by Nanouz** [comment]
What does that mean exactly? It is related to uploading the file? I dont use any XML I know of.

**Comment by Nanouz** [comment]
This is an additional error : The character encoding of the plain text document was not declared. The document will render with garbled text in some browser configurations if the document contains characters from outside the US-ASCII range. The character encoding of the file needs to be declared in the transfer protocol or file needs to use a byte order mark as an encoding signature. Does that tell you something more? Sorry to bother you with this.

**Comment by Nanouz** [comment]
I found it! The file image was too big to posted. Now I just need to set some webconfig options to allow bigger file uploads I guess?

**Comment by RehanSaeed** [comment]
Yes, that would be controlled by web.config. Glad you found the answer.
